WalletWasabi.Backend/Controllers/ChaumianCoinJoinController.cs

```diff
@@ -261,6 +255,17 @@ public async Task<IActionResult> PostInputsAsync([FromBody, Required]InputsReque
 						inputs.Add(new Coin(inputProof.Input, txOut));
 					}
 
+					if (!allInputsConfirmed)
+					{
+						// Check if mempool would accept a fake transaction created with the registered inputs.
+						// Fake outputs: 200 + 400 because the size of a bech32 input is rougly the size of an outputs and this'd be a 200input + 400 output tx and 0.9/600=0.0015, which is well above the dust, so should be good.
```

Comment:
Or "the number of outputs".

Reply:
I got your point. You want to simulate the byte size of a real CoinJoin with the addition of these inputs. It could be good. Of course, this will also tech the descendant limit.

Reply:
Fixed